makefiles/dependency_resolution.inc.mk

```diff
@@ -93,4 +93,12 @@ else
                             "don't run this on public networks!$(COLOR_RESET)" 1>&2)
     endif
   endif
+
+  # Warn about PSA Crypto
+  ifneq (,$(filter psa_crypto,$(USEMODULE)))
+    $(shell $(COLOR_ECHO) "$(COLOR_YELLOW) You are going to use the PSA Crypto module,"\
+                          "which is only partly implemented and not yet thouroughly tested.\n"\
+                          "Please do not use this module in production, as it may introduce"\
+                          "security issues!" 1>&2)
```

Comment:
Is there anywhere documented what needs to be done so we can remove this warning?

Reply:
Not yet, I'll put that on my todo list.
How is this usually done?